examples/minimal-solidstart/src/utils/uploadthing.ts

```diff
@@ -2,7 +2,9 @@ import { generateComponents, generateSolidHelpers } from "@uploadthing/solid";
 
 import type { OurFileRouter } from "~/server/uploadthing";
 
+const url = "http://localhost:9898";
```

Comment:
nit: maybe add a comment here calling out that it only works in dev (even though it should be pretty obvious)